src/draw/lv_draw_label.h

```diff
@@ -77,6 +77,7 @@ typedef struct {
 typedef enum {
     LV_DRAW_LETTER_BITMAP_FORMAT_A8,
     LV_DRAW_LETTER_BITMAP_FORMAT_IMAGE,
+    LV_DRAW_LETTER_BITMAP_FORMAT_VECTOR,
```

Comment:
BITMAP VECTOR is kind weird.

Reply:
sure

Reply:
I have changed it to `LV_DRAW_LETTER_GLYPH_FORMAT_A8 ` in 
https://github.com/lvgl/lvgl/pull/5056/files#diff-ebd8930142aae725f59a3fc797b3fa540cdca53ec7fe17e614d1056ed8daf84e 
Please check if it's suitable.